draftfast/orm.py

```diff
@@ -58,7 +58,7 @@ def __contains__(self, player):
                 if p.name == player or p.short_name == player:
                     return True
         elif isinstance(player, Player):
-            return p in self.players
+            return self.players
```

Comment:
@sharkiteuthis Not sure if you wrote this piece but assuming this change was the intent?

Reply:
__contains__() is supposed to return a boolean, I think this change will break NFL if the string comparison short circuit doesn't return true

Reply:
I think the issue here is `p` was out of scope originally. Maybe this?:

```python
    def __contains__(self, player):
        if isinstance(player, str):
            for p in self.players:
                if p.name == player or p.short_name == player:
                    return True
        elif isinstance(player, Player):
            return player in self.players
        else:
            raise NotImplementedError
```